### ISSUE TITLE
Dashboard - Override when hover on the maps 

### ISSUE THREAD

**Issue by NguyenTrang091296** [aesirxio/pim-app#8]
Actual result:
![image](https://user-images.githubusercontent.com/39818849/210473650-5bc71002-bd80-472e-bafb-6ca9e6f50af8.png)

**Comment by lklong709** [comment]
@TrucNguyenx any solution for this case, should we add a background for this ?

**Comment by TrucNguyenx** [comment]
@lklong709 yeah, please add a background with white color opacity 0.9 and and shadow
![image](https://user-images.githubusercontent.com/118497336/211451026-ba4743bb-d899-4f1f-b96f-4407fdc8bf23.png)
cc @NguyenTrang091296
